cmd/libs/go2idl/client-gen/generators/generator-for-group.go

```diff
@@ -58,11 +58,11 @@ func (g *genGroup) GenerateType(c *generator.Context, t *types.Type, w io.Writer
 		"Group":                      namer.IC(g.group),
 		"types":                      g.types,
 		"Config":                     c.Universe.Get(types.Name{Package: pkgUnversioned, Name: "Config"}),
-		"DefaultKubernetesUserAgent": c.Universe.Get(types.Name{Package: pkgUnversioned, Name: "DefaultKubernetesUserAgent"}),
+		"DefaultKubernetesUserAgent": c.Universe.GetFunction(types.Name{Package: pkgUnversioned, Name: "DefaultKubernetesUserAgent"}),
```

Comment:
Instead of Get, GetFunction, and GetVariable, how about Type(), Function(), Variable(), and Const() (if needed)?